### ISSUE TITLE
Notebook on data summary

### ISSUE THREAD

**Issue by PetalsOnWind** [PetalsOnWind/Statistics-and-Econometrics-for-Data-Science#12]

**Comment by NirvikC** [comment]
I would like to work on this notebook. Please describe what I have to do about data summary since I am new to open source.

**Comment by PetalsOnWind** [comment]
@NirvikC Learn how to read data from a csv file to a dataframe (`pd.read_csv`). If you run `df.describe()` on a dataframe named `df` then you get a summary table. In this notebook, you have to come up with methods to generate those statistics (mean, median, 25%,75% percentiles, max, min, etc) from scratch.

**Comment by NirvikC** [comment]
Can I do it with any dataset and in jupyter notebook?

**Comment by PetalsOnWind** [comment]
@NirvikC Yes

**Comment by stormfireuttam** [comment]
Hello there, I have some prior experience working with Matplotlib and Pandas and would like to contribute to your project. Could you suggest me from where to start.

**Comment by PetalsOnWind** [comment]
@stormfireuttam I have explained what to do in this issue on the above comment. Let me know if you have some other query

**Comment by stormfireuttam** [comment]
[Pull Request #29 ](https://github.com/PetalsOnWind/Statistics-and-Econometrics-for-Data-Science/pull/29)

@PetalsOnWind  I have made a pull request regarding the task allotted for data summary.

**Comment by NirvikC** [comment]
@PetalsOnWind I have also tried to make a pull request regarding the task. Please check whether the pull request was made properly or not

**Comment by PetalsOnWind** [comment]
@NirvikC It was not made

**Comment by stormfireuttam** [comment]
@PetalsOnWind could you please tell whether my pull request will be considered or not..

**Comment by NirvikC** [comment]
@PetalsOnWind I have tried again, can you please check?

**Comment by NirvikC** [comment]
I did the required changes in the program( using numpy to find instead of pd library) and made a pull request but could not find how to upload on notebook folder,

**Comment by PetalsOnWind** [comment]
@NirvikC Do not open new PR. Continue committing to existing PRs

**Comment by NirvikC** [comment]
@PetalsOnWind You have told me to comment the codes more thoroughly , so does it mean more comments in the code?

**Comment by PetalsOnWind** [comment]
@NirvikC Yes. Look at the file on ANOVA by PriyaJ28. I think she did a good job with that. It'll give you better idea about how to comment.

**Comment by NirvikC** [comment]
@PetalsOnWind I tried to make required changes and made pull request, please check

**Comment by anjiii-18** [comment]
Can you assign me this for GSSoC21 ?

**Comment by sakshi012000** [comment]
can you please assign this to me for GSSoC21?

**Comment by skybluee2001** [comment]
I want to work on this issue under GSSOC'2021. Please assign this to me!!

**Comment by skybluee2001** [comment]
@PetalsOnWind, thanks for assigning me this issue. 
I would like to contribute a notebook containing the implementation & a detailed explanation of the Linear Regression model.

**Comment by skybluee2001** [comment]
@PetalsOnWind, thanks for assigning me this issue.

**Comment by anjiii-18** [comment]
@PetalsOnWind Thanks a lot for assigning me this issue. Can you please brief a little what are we exactly supposed to do.
Can we take any data set?

**Comment by sakshi012000** [comment]
> @NirvikC My comments on your PR have not been resolved. At least the PR shows no update

thanks for assigning me this issue.

**Comment by sakshi012000** [comment]
@PetalsOnWind thanks for assigning me this issue.

**Comment by PetalsOnWind** [comment]
@anjiii-18 Yes, take any dataset. Explain calculating summary statistics and p-values associated

**Comment by anjiii-18** [comment]
@PetalsOnWind I have uploaded the notebook. Kindly review it. The pull request is Statistical Notebook #121.

**Comment by skybluee2001** [comment]
@PetalsOnWind I have made a PR. Kindly review it. 
Pull request #123

**Comment by sakshi012000** [comment]
@PetalsOnWind I have made a PR. Kindly review it.
Pull request #124

**Comment by skybluee2001** [comment]
@PetalsOnWind, I have changed the images to LaTeX text. Kindly review it.
PR #123

**Comment by sakshi012000** [comment]
@PetalsOnWind  I have added references. Please review it. PR #124

**Comment by anjiii-18** [comment]
@PetalsOnWind I have done the required changes, and uploaded two notebooks. Kindly review it. PR #121

**Comment by anjiii-18** [comment]
@PetalsOnWind 
Kindly review both the notebooks .
Statistical & Probability Notebook : Part 1
Statistical & Probability Notebook : Part 2
pr #121

**Comment by anjiii-18** [comment]
I have added overall headings to notebook. Kindly review. :) pr#121
